test002.c: 函数 `f' :
test002.c:17: エラー: `p' に対する代入演算子が不正です.
test002.c:18: エラー: `p' に対する代入演算子が不正です.
test002.c:19: エラー: `p' に対する代入演算子が不正です.
